pms/models/pms_room_type.py

```diff
@@ -209,6 +209,6 @@ def create(self, vals):
     def get_room_type_capacity(self, pms_property_id):
         self.ensure_one()
         capacities = self.room_ids.filtered(
-            lambda r: r.pms_property_id.id == pms_property_id.id
+            lambda r: not r.pms_property_id or r.pms_property_id.id == id
```

Comment:
I don't understand the value of the "id" variable in the last comparison, can you use id directly because it is an ensure_one or is it a bug?

Reply:
It's a bug, I'll look into it next week. Well spotted 👌

Reply:
That should work better.